R/mainTests.r

```diff
@@ -2886,13 +2886,14 @@ filter_group_level <- function(data, meta) {
   return(output)
 }
 
-# filter_group_not_filter -------------------------------------
+# filter_group_is_filter -------------------------------------
 # Checking that filter groups are not filters
 
-filter_group_not_filter <- function(meta) {
+filter_group_is_filter <- function(meta) {
+  standard_cols_allowed <- c("school_name")
```

Comment:
Trying to follow the logic on `school_name` being a special case and can't work it out, I blame the sleep deprivation - what's your thinking on this?